### PR TITLE
Updated 2 workflows and tutorials to use the latest version of Galaxy-ML

### PR DESCRIPTION
**Pull request by kxk302**
# Thank you for contributing to the GTN! :yellow_heart:

## Please check that:

- [ ] Your pull request has a **good title**, e.g.
    - "Fix typo in ansible-galaxy tutorial"
    - "Add new transcriptomics tutorial covering a new sequencing technology"
- [ ] Please **replace this entire pull request message** with a *description of your changes*, it will help us review your pull request faster
- [ ] Check that your images are allowed to be re-hosted by the GTN

Once these are done, you're ready to go!

<details>
    <summary>Consider sustainable computing with Draft Mode :green_heart:</summary>

Is your pull request finished? 100% ready to be merged? Or do you want some time to work on it first (which we encourage! It's great to have visibility)

If not, then please consider creating this pull request as a draft.

Regular pull requests will trigger automated test runs when they are created
and every time you update them.

As a small contribution to sustainable computing, we skip the most energy intensive of these tests for pull requests marked as drafts. Once your pull request is ready to go, then you can click a button saying "Ready for Review" and the tests will be run!
</details>

### CODE REVIEW

**Comment by kxk302** [comment]
Hi @anuprulez,
I updated 2 workflows/tutorials and created this PR. When you get a chance, could you please take a look?
Thanks!

**Comment by bgruening** [comment]
@kxk302 are you up to generating a workflow test for this one?

**Comment by kxk302** [comment]
> @kxk302 are you up to generating a workflow test for this one?

@bgruening Sure! Is there a code snippet I could refer to? Thanks

**Comment by bgruening** [comment]
This here might help: https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html

File: topics/statistics/tutorials/RNN/workflows/Intro_To_RNN_v1.0.10.0.ga [hunk 1]
```diff
@@ -0,0 +1,368 @@
+{
+    "a_galaxy_workflow": "true",
```
Comment (hexylena) [inline topics/statistics/tutorials/RNN/workflows/Intro_To_RNN_v1.0.10.0.ga:2]: fyi there are more lintung errors which you'll find in the Files tab of GitHub. Thanks for fixing them!

**Comment by kxk302** [comment]
Seems all the checks have passed. Ready for review!

**Review by hexylena — APPROVED** [review]
Looks great, thank you for fixing all the linting complaints @kxk302 !

**Pull request merged**